### ISSUE TITLE
No more coloring syntax when using Yarn Spinner Loom in VS Code

### ISSUE THREAD

**Issue by DX5536** [TranquilMarmot/YarnLoom#21]
**Describe the bug**
When editing the node, the syntax are not colored properly like using just only Yarn Spinner.

**Expected behavior**
Expectation: Syntax to be colored (like the demo GIFs and using only Yarn Spinner).

**Actual behavior**
A clear and concise description of what actually happened.
Actually: The text are all reduced to one color (except for comments)

**Steps To Reproduce**

0. Install Yarn Spinner.
1. Install Yarn Spinner Loom.
2. Create a node and write my story.
3. Syntax colors stay the same instead of being colored differently.

**Screenshots**
Images can be directly drag-and-dropped into the GitHub issue editor to upload them to GitHub.
With Loom:
![Code_2022-06-03_18-35-50](https://user-images.githubusercontent.com/49788345/171910940-858f3a2f-299c-49f8-9138-055509db56a4.png)
Without Loom:
![Code_2022-06-03_18-37-48](https://user-images.githubusercontent.com/49788345/171910980-a5eaefcd-4a36-47c4-94d1-7d00c4e18162.png)

**Additional context**

**Environment**

- OS:  Window
- Visual Studio Code Version: 1.67.2
- Yarn Loom Extension Version: v.1.2.0

**Comment by nemmm** [comment]
Yeah I wanted to use Loom too but its quite outdated, link arrows are not working neither.

**Comment by TranquilMarmot** [comment]
I think this is an issue with having both the Yarn Spinner extension and Yarn Spinner Loom installed at the same time. They step on each other when trying to add syntax highlighting.

This extension is definitely outdated at this point, and I haven't had time to keep developing it. When I first made this the official extension didn't have a full-fledged language server and visual editor, so Loom made a lot more sense.

These days I would say it's best to stick with the official extension. At some point in the future I may try and pull the visual editor out of this and contribute it to the official extension since it does add a lot of nice features.
